master/internal/command/command.go

```diff
@@ -345,10 +345,16 @@ func (c *command) Receive(ctx *actor.Context) error {
 		ctx.Self().Stop()
 
 	case job.SetGroupWeight:
-		ctx.Respond(c.setWeight(ctx, msg.Weight))
+		err := c.setWeight(ctx, msg.Weight)
+		if ctx.ExpectingResponse() {
+			ctx.Respond(err)
```

Comment:
does the error get logged anywhere?

Reply:
I'll add it. The intended behavior was to bubble it back up to the user as part of job modification.

Reply:
`ctx.Log().WithError()` logs nothing. need to actually call info/warn/etc.. also, we can just do
```
if err != nil {
 	ctx.Log().WithError(err).Info("setting command job weight")
         if ctx.ExpectingResponse() {
 	        ctx.Respond(err)
         }
 }
```
if you want because the `ctx.Respond(nil)` is implied if the caller is expecting a response.